lib_nbgl/doc/mainpage_nanos.dox

```diff
@@ -0,0 +1,65 @@
+#ifndef HAVE_SE_TOUCH
```

Comment:
The file is not referenced from `doc/mainpage.dox`, is not it? 

Is it supposed to be a conditional inclusion protected with `#ifdef HAVE_SE_TOUCH`? (then we need to do so and remove `(Stax only)` from `doc/mainpage.dox`? And also to add `HAVE_SE_TOUCH` to PREDEFINED section of `doc/Doxyfile` to build the documentation for Stax by default? 

Or maybe it's better to have 2 `Screen drawing API` sections  - one for Stax and one for Nano and to build all documentation unconditionally?